### ISSUE TITLE
Octave fit issues 

### ISSUE THREAD

**Issue by agahkarakuzu** [qMRLab/qMRLab#130]
**1) GUI based problems. Do you encounter the same problem too?**   @tanguyduval 

**Problem: gui_mainfcn' undefined** 

I cannot get output with with the following:

```
Custom_OptionsGUI(Model);
Model = getappdata(0,'Model');
```

**2) error: octave_base_value::save_ascii(): wrong type argument 'object'**

I cannot trace the origin of this error, but might be problematic for TRAVIS.

**Comment by tanguyduval** [comment]
Yes, Octave is not compatible with the GUI. Copy the lines at the beginning of qMRLab.m (if moxunit_tools_platformisoctave)

**Comment by tanguyduval** [comment]
2) Yes, that is why we should use saveobj #127 , and save a structure with all properties in `Models`.

**Comment by agahkarakuzu** [comment]
NoiseLevel Octave compatibility issue:

```
error: argument 'RICIAN' is not a valid parameter
error: called from
    error at line 539 column 7
    parse at line 463 column 13
    scd_noise_fit_histo at line 16 column 1
    fit at line 66 column 28
    FitData at line 138 column 9
    NoiseLevel_batch at line 69 column 12

```

**Comment by tanguyduval** [comment]
I think it's because `RICIAN` is in capital letters, and octave does not manage that. #123

**Comment by agahkarakuzu** [comment]
Resolved #123.
